src/service/mockup/feedServiceMock.ts

```diff
@@ -0,0 +1,72 @@
+import { Article, Category, Feed, User } from "skiosa-orm";
+import { Arg, FieldResolver, Int, Query, Resolver, Root } from "type-graphql";
+import { PaginationArg } from "../../model/paginationArg";
+import { FeedService } from "../feedService";
+import { MockService } from "./mockService";
+
+@Resolver((_of) => Feed)
+export class FeedServiceMock
+  extends MockService
+  implements FeedService {
+  categories(feed: Feed, paginated?: PaginationArg): Promise<Category[]> {
+    throw new Error("Method not implemented.");
```

Comment:
Was this even reviewed?

Reply:
Wasn't in the ACs, so it didn't get looked at. The mock service is almost obsolete by the initDB script anyway.